tools/chunk/src/main/scala/org/allenai/nlpstack/chunk/OpenNlpChunker.scala

```diff
@@ -1,12 +1,15 @@
 package org.allenai.nlpstack.chunk
 
 import org.allenai.common.Resource
-import org.allenai.nlpstack.core.{ChunkedToken, Chunker, PostaggedToken}
+import org.allenai.nlpstack.core.{ ChunkedToken, Chunker, PostaggedToken }
 
-import opennlp.tools.chunker.{ChunkerME, ChunkerModel}
+import opennlp.tools.chunker.{ ChunkerME, ChunkerModel }
 
 class OpenNlpChunker extends Chunker {
   private val chunker = new ChunkerME(OpenNlpChunker.model)
+  private final val chunker_t: ThreadLocal[ChunkerME] = new ThreadLocal[ChunkerME]() {
+    override protected def initialValue(): ChunkerME = new ChunkerME(OpenNlpChunker.model)
+  }
```

Comment:
We don't need the old `chunker` variable anymore, right? Can you remove it, and rename the thread-local one so it's just called `chunker`? We don't use underscores in variable names in Scala.